docs/data/getting-started/overview/overview.md

```diff
@@ -0,0 +1,37 @@
+---
+title: MUI X - Overview
```

Comment:
Should we link to this here: https://mui.com/x?

Currently it links to data grid:
<img width="498" alt="Screenshot 2022-05-06 at 13 29 02" src="https://user-images.githubusercontent.com/13808724/167123637-e0189bca-8bc1-4ed2-b29a-b0d6f2fd9bb4.png">

Reply:
Good call!

Reply:
After reviewing some Core docs today, I'm realizing there are quite a few places where we probably want to insert this link. Maybe I should put together a separate PR after this one is merged to comb through outdated links?